src/fluid/fluid.cpp

```diff
@@ -207,6 +208,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   physics->AddField(p::entropy::name(), mprim_scalar);
   physics->AddField(p::cs::name(), mprim_scalar);
   physics->AddField(diag::ratio_divv_cs::name(), mprim_scalar);
+  physics->AddField(diag::localization_function::name(), mprim_scalar);
```

Comment:
we're not using the localization function right? Let's comment this out for now and only add it when we need it.